scripts/jenkins-mender-convert-build.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+
+LINARO_COMPILER=gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
+COMPRESSED_COMPILER=${LINARO_COMPILER}.tar.xz
+LINARO_BINARY_PATH=${LINARO_COMPILER}/bin
+
+DEBIAN_IMAGE=bone-debian-9.4-iot-armhf-2018-06-17-4gb.img
```

Comment:
Keep in mind that it is the last image of Bone Debian that works out of the box. In v9.5. we currently have a workaround that replaces MLO & u-boot.img files with those extracted from v9.4.
Finding solution to https://tracker.mender.io/browse/MEN-2404 would require some changes in `mender-convert` tool and it this script as welll.